exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
```

Comment:
Thats right, it uses the local version. Reason being that if you have the repository locally, it is reasonable to believe that you are developing it and want to use your latest changes.